### PR TITLE
Pull accredited lists from USDSTO

### PR DESCRIPTION
**Pull request by adamdossa**
Added `getAccreditedData` to return accredited & non-accredited investor data

### CODE REVIEW

**Review by satyamakgec — APPROVED** [review]
Changes Look good!!

File: contracts/modules/STO/USDTieredSTOStorage.sol [hunk 1]
```diff
@@ -34,6 +34,15 @@ contract USDTieredSTOStorage {
         uint256 mintedDiscountPoly;
     }
 
+    struct Investor {
+        // Whether investor is accredited (0 = non-accredited, 1 = accredited)
+        uint8 accredited;
+        // Whether we have seen the investor before (already added to investors list)
+        uint8 seen;
+        // An override for nonAccreditedLimitUSD (0 = no override)
```
Comment (F-OBrien) [inline contracts/modules/STO/USDTieredSTOStorage.sol]: Suggested description to clarify the 10**18 multiple
```suggestion
        // Overrides for default limit in USD for non-accredited investors multiplied by 10**18 (0 = no override)
```

File: contracts/modules/STO/USDTieredSTOStorage.sol [hunk 2]
```diff
@@ -66,17 +75,21 @@ contract USDTieredSTOStorage {
...
     mapping (address => bool) public usdTokenEnabled;
 
+    // List of all addresses that have been added as accredited or non-accredited without
+    // the default limit
+    address[] public investorsList;
+
     // Default limit in USD for non-accredited investors multiplied by 10**18
     uint256 public nonAccreditedLimitUSD;
 
     // Overrides for default limit in USD for non-accredited investors multiplied by 10**18
```
Comment (F-OBrien) [inline contracts/modules/STO/USDTieredSTOStorage.sol]: This commented code can also be removed

Comment (adamdossa) [reply contracts/modules/STO/USDTieredSTOStorage.sol]: @F-OBrien which code? I think only the comments are commented here.

Comment (F-OBrien) [reply contracts/modules/STO/USDTieredSTOStorage.sol]:
```
// Overrides for default limit in USD for non-accredited investors multiplied by 10**18
    /* mapping (address => uint256) public nonAccreditedLimitUSDOverride; */
```

Comment (F-OBrien) [reply contracts/modules/STO/USDTieredSTOStorage.sol]: sorry looks like I had put my comment against the line above this

Comment (adamdossa) [reply contracts/modules/STO/USDTieredSTOStorage.sol]: Thanks - done.

**Pull request merged**